doc/classes/NavigationServer3D.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationServer3D" inherits="Object" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Server interface for low-level 3D navigation access.
+		A server interface for low-level 3D navigation access.
 	</brief_description>
 	<description>
-		NavigationServer3D is the server responsible for all 3D navigation. It handles several objects, namely maps, regions and agents.
+		NavigationServer3D is the server responsible for all 3D navigation. It handles navigation maps, regions and agents.
```

Comment:
NavigationServer is only responsible for navigation mesh based navigation and agent avoidance.
E.g. it does not handle point based navigation from the AStar2d/3D/Grid classes at all.